src/cloudant/__init__.py

```diff
@@ -13,10 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-_cloudant_
-
-Cloudant / CouchDB Python Client API
-
+Cloudant / CouchDB Python client library API package
```

Comment:
Added a compatibility section to the docs.  Done in 743823d273eb1dfb8d9efbefaa169e302ac40894.